src/View/Components/Icon.php

```diff
@@ -2,38 +2,13 @@
 
 namespace WireUi\View\Components;
 
-use Illuminate\View\Component;
+use WireUi\Heroicons;
 
-class Icon extends Component
+class Icon extends Heroicons\Icon
```

Comment:
I prefer to use it like this:

```
use WireUi\Heroicons\Icon as BaseIcon;

class Icon extends BaseIcon
```

Reply:
Avoid using class aliases, prefer the namespace instead. 
Class aliases are really bad for code tracking and refactorations.